resources/js/components/editor/QuillEditor.vue

```diff
@@ -65,18 +67,11 @@
 
         data() {
             return {
-                storeState: store.state,
                 editor: null,
                 trans: JSON.parse(Canvas.lang)
             }
         },
 
-        watch: {
-            'storeState.form.body'(val) {
-                this.update();
-            }
-        },
-
```

Comment:
This piece is an important one because it only allowed for the post to save when something actually happened. When the component loads up, it fills the editor with any body content available, emitting a `text-change` event. Now, when you load up any existing post, it'll automatically save without any actions being done to it.

There needs to be a way to load the existing form body in but not save until actions are performed on it.